CONTRIBUTING.md

```diff
@@ -205,8 +205,8 @@ To write such an update:
      (good: "Implemented..", bad: "Started working on.."); use the backlog and
      calendars to collect
 
-   - **What are the goals of next week**: a short look-out onto the backlog and/or
-     roadmap; ask each contributor what they would like to get done next week
+   * **What are the goals of next week**: a short look-out onto the backlog and/or
```

Comment:
I think mixing `-` and `*` for bullets it not picked up well by markdown (the rich diff looks weird).
```suggestion
   - **What are the goals of next week**: a short look-out onto the backlog and/or
```